compiler/version_check.go

```diff
@@ -19,6 +21,9 @@ const GoVersion = 16
 // at goroot, and reports an error if it's not compatible
 // with this version of the GopherJS compiler.
```

Comment:
Consider documenting the change to CheckGoVersion's behavior here.

Perhaps it's also worth including at https://github.com/gopherjs/gopherjs#environment-variables.

Reply:
Good point, done. I'll also add it to the developer instructions in the wiki once the PR is merged.